src/pages/gateway/command-reference.md

```diff
@@ -7,10 +7,48 @@ description: A description of the CLI commands available for API Mesh for Adobe
 
 The API Mesh for Adobe Developer App Builder CLI allows you to manage and modify meshes. This page covers commands exclusive to the API Mesh. For authorization and other Adobe I/O Extensible CLI commands, refer to the [Adobe IO CLI command list]. For installation instructions, refer to [Getting Started].
 
+## aio api-mesh:init
+
+Creates a [local development environment](./developer-tools.md#initiate-a-local-environment). You only need to run this command if you want to set up a local environment.
+
+### Usage
+
+```bash
+aio api-mesh:init <project-name>
+```
+
+### Flags
+
+The following arguments are all optional. If you do not supply them, the terminal response will prompt you for the information.
+
+`-p` or `--path` allows you to specify the location to set up the local environment.
+
+`-g` or `--git` is a binary argument that requires `Y` or `N` to determine if you want to use `git` for your local environment.
+
+`-m` or `--packageManager` is a binary argument that requires `npm` or `yarn` to determine which package manager to use for the local environment.
```

Comment:
In the Developer Tools topic, you linked to npm/yarn. I recommend doing so here too.